platforms/documentation/docs/src/docs/userguide/authoring-builds/structuring/composite_builds.adoc

```diff
@@ -135,11 +135,33 @@ include::sample[dir="snippets/developingPlugins/testingPlugins/groovy/include-pl
 Most builds can be included in a composite, including other composite builds.
 There are some restrictions.
 
-Every included build:
+In a regular build, Gradle ensures that each project has a unique _project path_.
+It makes projects identifiable and addressable without conflicts.
 
-* Must not have a `rootProject.name` the same as another included build.
-* Must not have a `rootProject.name` the same as a top-level project of the composite build.
-* Must not have a `rootProject.name` the same as the composite build `rootProject.name`.
+In a composite build, Gradle adds additional qualification to each project from an included build to avoid project path conflicts.
+The full path to identify a project in a composite build is called a _build-tree path_.
+It consists of a _build path_ of an included build and a _project path_ of the project.
+
+By default, build paths and project paths are derived from directory names and structure on disk.
+Since included builds can be located anywhere on disk, their build path is determined by the name of the containing directory.
+This can sometimes lead to conflicts.
+
+To summarize, the included builds must fulfill these requirements:
+
+* Each included build must have a unique build path.
+* Each included build path must not conflict with any project path of the main build.
+
+These conditions guarantee that each project can be uniquely identified even in a composite build.
+
+If conflicts arise, the way to resolve them is by changing the _build name_ of an included build:
+
+.settings.gradle.kts
+[source,kotlin]
+----
+includeBuild("some-included-build") {
+    name = "other-name"
```

Comment:
🤔 I don't remember, is there a test that show that this works?

Reply:
I think this test checks that
https://github.com/gradle/gradle/blob/1f67db00d08ff7bfd42cba32826fff7c9ee9a027/subprojects/composite-builds/src/integTest/groovy/org/gradle/integtests/composite/IncludedBuildValidationIntegrationTest.groovy#L113